src/Aeon/Symfony/AeonBundle/Validator/Constraints/NotHoliday.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Aeon\Symfony\AeonBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class NotHoliday extends Constraint
```

Comment:
I guess I will also add `Holiday` just in case someone would actually like to check if given date is a holiday ☺️